### ISSUE TITLE
Nipple is solid color, how do i fix ? 

### ISSUE THREAD

**Issue by HugglesGamerGirl** [FlailingFog/KK-Blender-Porter-Pack#315]
due to its NSFW nature this is never touched upon on the tutorial and i expected to easily be able to figure it out on my own by playing around in the NSFW shader section but sadly haven't figured it out how

Let me explain this issue: due to the fact I'm unsure if i can send NSFW screenshots of the nipples here ill try my best to describe what is going on, basically instead of being detailed as you would see in the Koikatsu editor once imported into blender it's just a solid color now i know that the texture is still here somewhere because when i played with the FAC sliders (shown below)
![Screenshot 2022-11-23 130912](https://user-images.githubusercontent.com/118823112/203618530-b487461a-6586-4269-aedb-393de2f68fc3.png)
i managed to get a glimpse of the texture used for nipples however the saturation is not correct

i was wondering what do i have to do to make the nipples texture look like how they do in Koikatsu

**Comment by FlailingFog** [comment]
The color is being added by default. This worked in previous versions but doesn't work now for some reason. If you turn down the top add slider it should look right
![image](https://user-images.githubusercontent.com/65811931/204428387-49c19aba-3bbb-42a4-b318-6aa8fb4633da.png)

You can also try these colors to use as a starting point (these can be copy pasted into the color input)
```
[1.0, 0.48, 0.48, 1.0]
[0.9, 0.0, 0.1, 1.0]
[1.0, 0.8, 0.8, 1.0]
[1.0, 0.5, 0.5, 1.0]
```

**Comment by FlailingFog** [comment]
Fixed in [e4c0e6e](https://github.com/FlailingFog/KK-Blender-Porter-Pack/commit/e4c0e6e49b15babbb1922cb4a8d295003d1b620b) I think
